### DISCUSSION TITLE
怎么将打包产物中的less转换成css

### DISCUSSION THREAD

**Discussion by Lands-1203** [umijs/father#748] [Q&A]
怎么将打包产物中的less转换成css

**Comment by fz6m** [comment]
father 只是 bundleless ，不会处理源代码里的非 TS 文件，只会拷贝到产物里对应位置，所有非 TS 源码的功能（如 less / css module 等），都需要引入这个库的主体项目来支持这些功能。

比如使用 antd 4 的时候，样式可以引入 `antd.less` ，但项目本身要支持 less 转译。如果需要 css 产物，应该选择 bundle 为单文件的打包器（如 father umd ，或 rollup 等）。
